docs/release-notes.txt

```diff
@@ -6,6 +6,39 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.7
+^^^^^^^^^^^^^^
+
+**Release Date:** June 11, 2020
+
+- **Breaking Change:** Gradient clipping for PyTorchTrial should now be specified via `determined.PyTorch.PyTorchCallback` via the `on_before_optimizer_step()` instead of being specified via the experiment configuration. Determined provides two pre-made callbacks for gradient clipping:  `determined.PyTorch.ClipGradsL2Norm` and `determined.PyTorch.ClipGradsL2Value`.
+
+- Add support for Keras callbacks that stop training by setting the model.stop_training flag to True, including the official EarlyStopping callback: https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/EarlyStopping. When a stop is requested, we will finish the training (or validation) step we are in, checkpoint, and terminate the trial.
+
+- Add support for Estimator hooks that stop training by calling run_context.request_stop(), including the official stop_if_no_decrease_hook: https://www.tensorflow.org/api_docs/python/tf/estimator/experimental/stop_if_no_decrease_hook. When a stop is requested, we will finish the training (or validation) step we are in, checkpoint, and terminate the trial.
```

Comment:
blcoking: ditto above, and monospace run_context.request_stop(), stop_if_no_decrease_hook

Reply:
line 18